terms.toml

```
@@ -2,6 +2,10 @@
 term = "Full Node"
 translation = "全節點"
 
+[[terms]]
+term = "Impermenant Loss"
```

Google suggests the following spelling 

```suggestion
term = "Impermanent Loss"
```

---

ahhhhh good catch 🤣